core/src/test/java/com/google/googlejavaformat/java/DiagnosticTest.java

```diff
@@ -26,8 +26,9 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import org.junit.Rule;
-import org.junit.Test;
+import java.util.Locale;
+
+import org.junit.*;
```

Comment:
I believe wildcard imports are disallowed in the Google Java Style guide. Have you run `google-java-format` on your changes already?